recipes/libsbml/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "libsbml" %}
+{% set version = "5.20.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sbmlteam/libsbml/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 02c225d3513e1f5d6e3c0168456f568e67f006eddaab82f09b4bdf0d53d2050e
+  
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('libsbml', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [not win]
+    - ninja  # [win]
+    - zlib
+    - bzip2
+    - libxml2
+    
+  host:
+    - zlib
+    - bzip2
+    - libxml2
+  
+test:
+  commands:
+    - test -f "${PREFIX}/lib/libsbml${SHLIB_EXT-.so}"  # [linux]
+    - test -f "${PREFIX}/lib/libsbml${SHLIB_EXT-.dylib}"  # [osx]    
+    - if not exist %PREFIX%/lib/libsbml.lib exit 1  # [win]
```

Comment:
.lib could either be static library or import library on Windows. Please switch to compiling shared library for Windows if this is a static library. An import library is a library stub that is used by the Windows compiler and it is not needed at runtime.